src/ansys/fluent/core/filereader/casereader.py

```diff
@@ -1,7 +1,5 @@
 # Compatibility aliases
-from .case_file import (  # noqa: F401
-    InputParameter,
-    OutputParameter,
-    _get_processed_string,
-)
 from .case_file import CaseFile as CaseReader  # noqa: F401
+from .case_file import InputParameter, OutputParameter  # noqa: F401
+from .case_file import SettingsFile as SettingsReader  # noqa: F401
```

Comment:
This is not needed as the other only exists for compatibility

Reply:
Keeping it now till OptiSlang uses the new files.